demisto_sdk/commands/prepare_content/preparers/marketplace_incident_to_alert_prepare.py

```diff
@@ -0,0 +1,52 @@
+import logging
+
+from demisto_sdk.commands.common.constants import MarketplaceVersions
+
+logger = logging.getLogger("demisto-sdk")
+
+
+class MarketplaceIncidentToAlertPreparer:
+
+    @staticmethod
+    def prepare(
+            data: dict,
+            marketplace: MarketplaceVersions = MarketplaceVersions.XSOAR,
+    ) -> dict:
+        """
+        Iterate over all the given content item description fields and if a description field contains the word
+        incident / incidents, then replace it with alert / alerts.
+        Args:
+            data: content item data
+            marketplace: Marketplace. Replace just for XSIAM Marketplace.
+
+        Returns: A (possibly) modified content item data
+
+        """
+        # Replace incidents to alerts only for XSIAM
+        if marketplace != MarketplaceVersions.MarketplaceV2:
```

Comment:
When uploading a content item supported in both xsoar & xsiam, we'll reach this part with `marketplace=MarketplaceVersions.XSOAR`. Then, we must check the descriptions as they may contain `<-incident->`s, which shouldn't appear in the final content item.